src/test/java/edu/tamu/cap/controller/VerifyRepositoryViewSettingsControllerTest.java

```diff
@@ -29,19 +33,17 @@
 import org.springframework.restdocs.payload.FieldDescriptor;
 import org.springframework.restdocs.request.ParameterDescriptor;
 import org.springframework.security.test.context.support.WithMockUser;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.web.servlet.MockMvc;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-
 import edu.tamu.cap.model.RepositoryView;
 import edu.tamu.cap.model.repo.RepositoryViewRepo;
 import edu.tamu.cap.service.RepositoryViewType;
 import edu.tamu.cap.service.repositoryview.FedoraService;
 import edu.tamu.cap.utility.ConstraintDescriptionsHelper;
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
`@SpringBootTest` already includes this, please remove `ExtendWith(SpringExtension.class)`.